lib/gui/app/components/svg-icon.js

```diff
@@ -108,9 +153,14 @@ class SVGIcon extends react.Component {
 SVGIcon.propTypes = {
 
   /**
-   * @summary SVG contents or path to the resource
+   * @summary Paths to SVG files to be tried in succession if any fails
+   */
+  paths: propTypes.array,
+
+  /**
+   * @summary List of embedded SVG contents to be tried in succession if any fails
    */
-  path: propTypes.string.isRequired,
+  contents: propTypes.array,
```

Comment:
Minor nit, but given that the code attempts to parse `contents` before parsing `paths`, maybe `contents` should be declared above `paths`?